pkg/machines/hostvmslist.jsx

```diff
@@ -28,6 +28,15 @@ import GraphicsConsole from './components/graphicsConsole.jsx';
 
 const _ = cockpit.gettext;
 
+function left_click(fun) {
```

Comment:
The rest of the file uses camelCase, I suggest `mouseClick` or something along those lines.